packages/itensor/quri_parts/itensor/circuit/__init__.py

```diff
@@ -0,0 +1,123 @@
+from collections.abc import Mapping
+
+import juliacall
+from juliacall import Main as jl
+
+from quri_parts.circuit import NonParametricQuantumCircuit, gate_names
+from quri_parts.circuit.gate_names import (
+    SingleQubitGateNameType,
+    ThreeQubitGateNameType,
+    TwoQubitGateNameType,
+    is_gate_name,
+    is_single_qubit_gate_name,
+    is_three_qubit_gate_name,
+    is_two_qubit_gate_name,
+)
+
+_single_qubit_gate_itensor: Mapping[SingleQubitGateNameType, str] = {
+    gate_names.Identity: "I",
+    gate_names.X: "X",
+    gate_names.Y: "Y",
+    gate_names.Z: "Z",
+    gate_names.H: "H",
+    gate_names.S: "S",
+    gate_names.Sdag: "Sdag",
+    gate_names.SqrtX: "√X",
+    gate_names.SqrtXdag: "SqrtXdag",
+    gate_names.SqrtY: "SqrtY",
+    gate_names.SqrtYdag: "SqrtYdag",
+    gate_names.T: "T",
+    gate_names.Tdag: "Tdag",
```

Comment:
Sorry, I don't know what you mean. 
Could you please tell me more about it?

Reply:
Maybe I should have commented on `library.jl`.
For example in `library.jl` `SqrtY` is defined as:

```julia
ITensors.op(::OpName"SqrtY", ::SiteType"Qubit") = [
    (1+im)/2 (-1-im)/2
    (1+im)/2 (1+im)/2
]
```

However, since `√X` is defined in ITensor, we may be able to follow the same convention and use `√Y` as the gate name instead of `SqrtY`.

Reply:
I see, thank you!